index.html

```diff
@@ -0,0 +1,239 @@
+                            <ul class="none">
+                                <li> <a class="nav_link black" href="#"><i class="fa fa-tag"></i>Telecomunication</a></li>
+                                <li><a class="nav_link black" href="#"><i class="fa fa-suitcase"></i>Expedition Growth Capital</a></li>
+                            </ul>
+                        </div>
+                        <div class="mb2">
+                            <p>Ringover raises $12 million to bring enterprise communications to the cloud on VentureBeat</p>
+
+                        </div>
+                    </div>
+                    <div class="cflex fund sfund">
+                        <img class="mb2" alt="ff" src="img/second.png">
+                        <div class="mb2">
+                            <ul class="none">
+                                <li><a class="nav_link black" href="#"><i class="fa fa-tag"></i> Biotechnologie</a></li>
+                                <li><a class="nav_link black" href="#"><i class="fa fa-suitcase"></i> VisVires New Protein</a></li>
+                            </ul>
+                        </div>
+                        <div class="mb2">
+                            <p>BREAKING: Food testing startup SwissDeCode nets $4.2m from VisVires New Protein, EIT Food on AgFunderNews</p>
+                        </div>
+                    </div>
+                    <div class="cflex fund sfund">
+                        <img class="mb2" alt="ff" src="img/third.png">
+                        <div class="mb2">
+                         <ul class="none">
+                             <li><a class="nav_link black" href="#"><i class="fa  fa-tag"></i>Internet</a></li>
+                            <li><a class="nav_link black" href="#"><i class="fa fa-suitcase"></i>Buyo Capital</a></li>
+                          </ul>
+                        </div>
+                        <div class="mb2">
+                            <p>SoftBank-backed travel platform Klook raises $200M amid COVID-19 on TechCrunch</p>
+                        </div>
+                    </div>
+                    <div class="cflex fund sfund">
+                        <img class="mb2" alt="ff" src="img/fourth.png">
+                        <div class="mb2">
+                            <ul class="none">
+                                <li><a class="nav_link black" href="#"><i class="fa fa-tag"></i>Mobile</a></li>
+                                <li><a class="nav_link black" href="#"><i class="fa fa-suitcase"></i>DST Global</a></li>
+                             </ul>
+                        </div>
+                        <div class="mb2">
+                            <p>Food delivery startup Wolt raises $530M after tripling revenues on SiliconANGLE</p>
+                        </div>
+                    </div>
+                </div>
+            </section>
+
+        </main>
+        <!--Footer-->
+        <footer class="cflex">
```

Comment:
## Improve the footer
- [ ] Be mindful of how the social icons have been styled. 
- [ ] Some images are also not loading in this section. Do you mind fixing them,?

|[Source Page](https://perma.cc/M5ZV-Q2D6)|[Clone Page](https://1v4n4.github.io/Responsive-web-page/)|
|:--:|:--:|
|![footer](https://user-images.githubusercontent.com/22345961/113048974-ca09b100-91ab-11eb-857a-e698b809840d.png)|![footer-clone](https://user-images.githubusercontent.com/22345961/113048988-cece6500-91ab-11eb-87b3-f4acaa77c0ee.png)|